src/i18n/locales/en/translation.json

```diff
@@ -567,25 +567,24 @@
         "status": "Check payment status"
       }
     },
-    "publishers": {
-      "used": {
-        "title": "Your ⚡️ Websites",
-        "description": "Websites where you have used Alby before",
-        "no_info": "No websites yet."
-      },
-      "suggestions": {
-        "title": "Other ⚡️ Websites",
-        "description": "Websites where you can use Alby",
-        "list": {
-          "trading": "Trading",
-          "gaming": "Gaming",
-          "entertaiment": "Entertaiment",
-          "shopping": "Shopping",
-          "miscellaneous": "Miscellaneous",
-          "showcases": "Showcases"
-        }
-      },
-      "publisher": {
+    "discover": {
```

Comment:
This will happen every now and then. Had another look but no success.
@rolznz opened this: https://github.com/WeblateOrg/weblate/issues/8558  
Let's see. At least it's added to their backlog. Wondering how other projects handle this.

Reply:
> This will happen every now and then. Had another look but no success. @rolznz opened this: [WeblateOrg/weblate#8558](https://github.com/WeblateOrg/weblate/issues/8558) Let's see. At least it's added to their backlog. Wondering how other projects handle this.

Please help them fix this. I like to translate Alby but it's not good have missing translations, discourages people like me.

Reply:
I guess currently the best chance we have is to check the git history: https://github.com/getAlby/lightning-browser-extension/commits/c8dc5c301e70c142388c061ac213158b922b5da1/src/i18n/locales/es/translation.json

Reply:
This MR is really old now and probably has many conflicts. If we want to go ahead I think it would be good to make a new one, and update all the other translation files to match the new key.

I am pretty disappointed by weblate's lack of automation (both for contributors and maintainers). Maybe I could work on a small tool for this ?!?